src/autoTune.ml

```diff
@@ -226,6 +226,8 @@ let focusOnSpecification () =
   (* TODO: Finish these two below later *)
   | ValidDeref
   | ValidMemtrack -> ()
+  | MemorySafety -> () (* TODO: This is here for now just to complete the pattern match *)
+  | ValidMemcleanup -> ()
```

Comment:
Should probably implement these as well to enable the required analyses because they're not default in the SV-COMP conf (and shouldn't be for other categories).

Reply:
The memory safety analyses should be now enabled in the auto-tuner: https://github.com/goblint/analyzer/pull/1201/commits/00cc9b541ae6e6270906c16cbba94498692ab6f3